designer/pydesigner.py

```diff
@@ -711,7 +715,35 @@
                 akc_out = img.akcoutliers()
                 img.akccorrect(akc_out)
             mk, rk, ak, kfa, mkt, trace = img.extractDKI()
+            if args.median:
+                if args.mask:
+                    brainmaskPath = op.join(outpath, 'brain_mask.nii')
+                    import nibabel as nib
```

Comment:
Why not just add this to the top? Since the import is named, it's unlikely to have a collision.